package.json

```diff
@@ -23,11 +23,12 @@
   "license": "MIT",
   "homepage": "https://bitbucket.org/trellisboston/react-components#readme",
   "dependencies": {
-    "embla-carousel-react": "^6.2.0",
     "next": "^12.1.0",
     "prop-types": "^15.8.1",
     "react": "^16.8.0 || ^17",
     "react-dom": "^16.8.0 || ^17",
+    "react-easy-swipe": "^0.0.22",
+    "react-icons": "^4.4.0",
```

Comment:
NPM shows this is 46.5MB unpacked! Why not take the SVG code for the icons we need from the Figma designs instead?